android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlReceiver.java

```diff
@@ -56,4 +76,43 @@ public void onReceive(Context context, Intent intent) {
     public String getSdlServiceName() {
         return SdlService.class.getSimpleName();
     }
+
+    private final BroadcastReceiver mUsbReceiver = new BroadcastReceiver() {
```

Comment:
Name should be more specific, and avoid the `m` member naming convention if possible.
```suggestion
    private final BroadcastReceiver usbPermissionReceiver = new BroadcastReceiver() {
```